.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: ruby
+sudo: false
+rvm:
+  - 2.0
+  - 2
+bundler_args: --without cookbooks
+cache: bundler
+gemfile: Gemfile
+before_install:
+  - chef --version &> /dev/null || curl -L https://www.getchef.com/chef/install.sh | sudo bash -s -- -P chefdk
```

Comment:
should we pin this to what is set to https://github.com/bloomberg/chef-bach/blob/master/build_bins.sh#L25 ?

Reply:
Fixed to use the same version for now; ideally master should float though.